modules/machine-user-infra-machines-iso.adoc

```diff
@@ -19,17 +19,39 @@ You can create more {op-system-first} compute machines for your bare metal clust
 
-. After the instance boots, press the `TAB` or `E` key to edit the kernel command line.
-. Add the parameters to the kernel command line:
+. Boot the {op-system} ISO image without specifying any options or interrupting the live boot sequence. Wait for the installer to boot into a shell prompt in the {op-system} live environment.
++
+[NOTE]
+====
+It is possible to interrupt the {op-system} installation boot process to add kernel arguments. However, for this ISO procedure you should use the `coreos-installer` command as outlined in the following steps, instead of adding kernel arguments.
+====
+
+. Run the `coreos-installer` command and specify the options that meet your installation requirements. At a minimum, you must specify the URL that points to the Ignition config file for the node type, and the device that you are installing to:
 +
 [source,terminal]
 ----
-coreos.inst.install_dev=sda <1>
-coreos.inst.ignition_url=http://example.com/worker.ign <2>
+$ sudo coreos-installer install --ignition-url=http://<HTTP_server>/<node_type>.ign <device> --ignition-hash=sha512-<digest> <1><2>
+----
+<1> You must run the `coreos-installer` command by using `sudo`, because the `core` user does not have the required root privileges to perform the installation.
+<2> The `--ignition-hash` option is required when the Ignition config file is obtained through an HTTP URL to validate the authenticity of the Ignition config file on the cluster node. `<digest>` is the Ignition config file SHA512 digest obtained in a preceding step.
++
+[NOTE]
+====
+If you want to provide your Ignition config files through an HTTPS server that uses TLS, you can add the internal certificate authority (CA) to the system trust store before running `coreos-installer`.
+====
++
+The following example initializes a bootstrap node installation to the `/dev/sda` device. The Ignition config file for the bootstrap node is obtained from an HTTP web server with the IP address 192.168.1.2:
++
+[source,terminal]
+----
+$ sudo coreos-installer install --ignition-url=http://192.168.1.2:80/installation_directory/bootstrap.ign /dev/sda --ignition-hash=sha512-a5a2d43879223273c9b60af66b44202a1d1248fc01cf156c46d4a79f552b6bad47bc8cc78ddf0116e80c59d2ea9e32ba53bc807afbca581aa059311def2c3e3b
 ----
-<1> Specify the block device of the system to install to.
-<2> Specify the URL of the compute Ignition config file. Only HTTP and HTTPS protocols are supported.
 
-. Press `Enter` to complete the installation. After {op-system} installs, the system reboots. After the system reboots, it applies the Ignition config file that you specified.
+. Monitor the progress of the {op-system} installation on the console of the machine.
++
+[IMPORTANT]
+====
+Be sure that the installation is successful on each node before commencing with the {product-title} installation. Observing the installation process can also help to determine the cause of {op-system} installation issues that might arise.
```

Comment:
Does `Be sure` mean `Ensure` here? If yes, ensure is a better word.

Reply:
Done